gapic/schema/wrappers.py

```diff
@@ -141,6 +143,15 @@ class MessageType:
     def __getattr__(self, name):
         return getattr(self.message_pb, name)
 
+    @utils.cached_property
+    def field_types(self) -> Sequence[Union['MessageType', 'EnumType']]:
+        """Return all composite fields used in this proto's messages."""
+        answer = []
+        for field in self.fields.values():
+            if field.message and not field.enum:
```

Comment:
I...do not know. That `if` statement also makes no sense because a field can never have both, so the `and not field.enum` is duplicative.

Reading this now, I believe enums should be included.